ui/lib/config-ui/addon/routes/messages/index.js

```diff
@@ -4,5 +4,34 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class MessagesRoute extends Route {}
+export default class MessagesRoute extends Route {
+  @service store;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+    },
+    authenticated: {
+      refreshModel: true,
+    },
+  };
+
+  async model(params) {
+    try {
+      const { authenticated, page } = params;
+      return await this.store.lazyPaginatedQuery('config-ui/message', {
+        authenticated,
+        responsePath: 'data.keys',
+        page: page || 1,
```

Comment:
this is the default set here. You want your queryParams to clear on tab switch, so don't set it by default, just return page 1 by default.